Tests/Unit/Controller/Admin/Order/OrderTabDescriptorTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Controller\Admin\Order\OrderTabDescriptor;
+use Wirecard\Oxid\Tests\Unit\Controller\Admin\TestDataHelper;
+
+class OrderTabDescriptorTest extends Wirecard\Test\WdUnitTestCase
+{
+    /**
+     * @var OrderTab
+     */
+    private $_orderTab;
+
+    protected function dbData()
+    {
+        return TestDataHelper::getDemoData();
+    }
+
+    public function testRender()
+    {
+        $_GET['oxid'] = 'oxid 1';
+        $this->_orderTabDescriptor = new OrderTabDescriptor();
+        $this->_orderTabDescriptor->render();
+
+        $aViewData = $this->_orderTabDescriptor->getViewData();
+
+        $this->assertArrayHasKey('emptyText', $aViewData);
+    }
+
+    /**
+     * @dataProvider getDataProvider
+     */
+    public function testGetData($sOrderId, $aExpectedArray)
+    {
+        $_GET['oxid'] = $sOrderId;
```

Comment:
Please use `$this->setRequestParameter()`. See above.

Reply:
done!